client/src/components/navbar.jsx

```diff
@@ -60,15 +60,14 @@ const Box = Styled.div`
 `
 
 class Navbar extends React.Component {
-  constructor() {
-    super()
+  constructor(props) {
+    super(props)
 
     this.state = {
       currentTab: 'materials',
       isSelected: true
     }
 
-    this.myRef = React.createRef();
     this.handlePick = this.handlePick.bind(this);
   }
 
```

Comment:
do you need your console.log in handlePick anymore?

Reply:
I went a different route on this. So deleted it in the next PR.